### DISCUSSION TITLE
Email Events

### DISCUSSION THREAD

**Discussion by debaduttar** [ocsf/ocsf-schema#377] [Q&A] [answered]
Hi,
I am new to OCSF and trying to understand the schema. 
How Email Delivery Finding event (Finding category) and Email Activity Event (System Activity category) are related? Do they exist independent of each other? Why they are not in the same category?

Apologies for the very basic question but will appreciate if I can get a answer👍 .

Regards
Deb

**Comment by ablyler** [comment, marked as answer]
The email finding classes got removed a few hours ago: https://github.com/ocsf/ocsf-schema/commit/c299fdc40dd2bb947a7bbda7a9ef8c6795f4992f

I am unsure when the schema website will be updated to reflect this, but the idea is to use the activity events and the malware profile if it contains an attack. Does this fit your use case?

**Comment by debaduttar** [comment]
Yes, it does clear my confusion and fits my use case. I can see Email related classes  in Network activity now. 
Many thanks.
